src/emails/templates/breachAlert/BreachAlertEmail.tsx

```diff
@@ -34,6 +35,7 @@ export const BreachAlertEmail = (props: Props) => {
     <mjml>
       <mj-head>
         <mj-preview>{l10n.getString("email-spotted-new-breach")}</mj-preview>
+        {HeaderStyles()}
```

Comment:
I think it's the same, but any reason not to do this? (Here and everywhere in this PR?)

```suggestion
        <HeaderStyles/>
```